Jenkinsfile

```diff
@@ -1,74 +1,275 @@
 max_time = 180
 
 stage("Unit Test") {
-  node('linux-gpu') {
-    ws('workspace/autugluon-py3') {
-      timeout(time: max_time, unit: 'MINUTES') {
-        checkout scm
-        VISIBLE_GPU=env.EXECUTOR_NUMBER.toInteger() % 8
-        sh """#!/bin/bash
-        set -ex
-        conda env update -n autogluon_py3 -f docs/build.yml
-        conda activate autogluon_py3
-        conda list
-        export CUDA_VISIBLE_DEVICES=${VISIBLE_GPU}
-        env
-        export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64
-        export MPLBACKEND=Agg
-        export MXNET_CUDNN_AUTOTUNE_DEFAULT=0
+  parallel 'core': {
```

Comment:
Since this is a really nice change that is separate from the vision task refactor logic, could this be moved into a separate PR so we can merge it in quickly?

Reply:
yes, see https://github.com/awslabs/autogluon/pull/773